sample-app/src/test/scala/example/ActorSpecBase.scala

```diff
@@ -12,15 +11,26 @@ import testkit.AkkaTypedSpanScaleFactorSupport
 /** A test class which improve consistency and reduce boilerplate.
   *
   * @see [[https://www.scalatest.org/user_guide/defining_base_classes Defining base classes for your project]]
-  * @todo Use Typed ActorSystem
   */
-abstract class ActorSpecBase(system: ActorSystem)
-    extends ScalaTestWithActorTestKit(system.toTyped)
+abstract class ActorSpecBase(testKit: ActorTestKit)
+    extends ScalaTestWithActorTestKit(testKit)
     with AnyWordSpecLike
     with Matchers
     with Inside
     with BeforeAndAfterAll
     with Eventually
     with ScalaFutures
     with EitherValues
-    with AkkaTypedSpanScaleFactorSupport
+    with AkkaTypedSpanScaleFactorSupport {
+
+  def this() = {
+    // デフォルトの振る舞いでは `application-test` もしくは `reference` のみが読み込まれる。
+    // テスト全般にわたって `application` と `reference` を使用したいため、ConfigFactory#load を使用する。
+    this(ActorTestKit(ConfigFactory.load()))
```

Comment:
> [Akka 2.6.8 - akka.actor.testkit.typed.scaladsl.ActorTestKit](https://doc.akka.io/api/akka/2.6.8/akka/actor/testkit/typed/scaladsl/ActorTestKit$.html#apply():akka.actor.testkit.typed.scaladsl.ActorTestKit)
> 
> Config loaded from application-test.conf if that exists, otherwise using default configuration from the reference.conf resources that ship with the Akka libraries. The application.conf of your project is not used in this case.

デフォルトの振る舞いは 上記のURL から確認できます。